### ISSUE TITLE
attribute support use case

### ISSUE THREAD

**Issue by geyang** [Matt-Esch/virtual-dom#108]
Hi @Raynos and all,

I read through your comments in #32 and #12, mostly searching for a way to allow attribute support when diff-patching a DOM element. Here is my use case:

I am using angular to make a contenteditable element. Inside the contenteditable, a few directives (webcomponent-like element) are used. I want to take advantage of the bindings already exisitng in the contenteditable, but allow differential update of the view so that caret recovery is easy, and components which tend to take longer to compile does not have to be updated all the time. 

One of the elements I am using is math-jax. The convention is to save the LaTeX code inside the `alt` attribute. I can not save this information inside of the element since then updating the code will require replacing the already rendered content of the element with raw code, leading to flashes. 

Please let me know if there is a good way to add support.

Thanks for the good work.
